files/sync.py

```diff
@@ -5,83 +5,42 @@
 import os
 import logging
 
-logg = logging.getLogger(__file__)
+# platform imports
+import config
 
+file_sync_logger = logging.getLogger(__file__)
 
-class FileSyncer:
-    """Parent class to be overloaded by component providing access to
-    external file resources.
 
-    A function is passed to the constructor by the implementer, which _must_ 
-    fulfull the following interface:
-        * accept a single string as argument for the filename to retrieve
-        * return an object implementing a read(blocksize=int) method
+def source_newer(last_commit_id):
```

Comment:
why renamed. now you are breaking the interface and s3 implementation will not work anymore.

Reply:
This is my brain at 6 AM having not slept all night, I have reverted the file syncer for now, then I will try and overload it as you intended in its design.